src/redux/reducers/loader.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState: { isLoading: boolean } = {
+  isLoading: false,
+};
+
+const loader = createSlice({
+  name: 'loader',
+  initialState,
+  reducers: {
+    show: (state): void => {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
can we do something with it ?
https://eslint.org/docs/rules/no-param-reassign#props